avm/utilities/pipelines/platform/Assign-Issue.ps1

```diff
@@ -0,0 +1,99 @@
+Function Get-AvmCsv {
+  [CmdletBinding()]
+  param (
+    [Parameter(Mandatory)]
+    [ValidateSet('Bicep-Resource', 'Bicep-Pattern', 'Terraform-Resource', 'Terraform-Pattern')]
+    [string]$ModuleIndex
+  )
+
+  # Retrieve the CSV file
+  if ($ModuleIndex -eq 'Bicep-Resource') {
+    try {
+      $unfilteredCSV = Invoke-WebRequest -Uri "https://aka.ms/avm/index/bicep/res/csv"
```

Comment:
Would you agree it might be a good idea to move all urls to the top of the script as variables? That way if they every change it would be easy to do so in one location

Reply:
Could be a good idea. But is the idea not to use the vanity URL's as these should not change.... However could be worth documenting that we use the Vanity URL's within the verbose logs?

Reply:
My point is just to move related, complicated static values to one location.

Reply:
Agree to use variables

Reply:
Also, can we check if we have the direct vanity link to the csv?

Reply:
Done. @matebarabas , do we have a vanity link to the CSV files?

Reply:
We do and you used it already :) 
![image](https://github.com/Azure/bicep-registry-modules/assets/5365358/12f5ae4c-55a8-4b57-b57f-8141a6d5c5d7)